website/content/en/docs/development-guide.md

```diff
@@ -39,6 +39,7 @@ make delete # Uninstall Karpenter
 ### Build and Deploy
 ```
 make dev                                  # build and test code
+kubectl create namespace karpenter        # create target namespace for deployment
```

Comment:
Nice catch. We typically have this namespace created as part of the getting-started guide (helm install --create-namespace).